### ISSUE TITLE
Add Containing Folder doesn't insert directly after selected file

### ISSUE THREAD

**Issue by harakiwi** [clsid2/mpc-hc#1535]
1.9.15 just works.

I can add files to my playlist and the mouse-over / hover playlist shows up and goes away as expected.

then the 1.9.16 broke adding files.

I'm used to drag one MP4 file to the playlist and use _Add containing folder_ to add the rest of the files. This doesn't work.

Clicking this option jumps around in the playlist and adds files to the bottom of the list. Not next to where the first file is in my playlist. 

Then the file names are somehow all lowercase.

So I reverted back to .15

.17, same problem

.18 same problem but now it also breaks the playlist.

I keep my playlist at the bottom of the screen, 20 entries long. Moving my mouse away from the playlist closed the playlist/hides the playlist in .15.

I stopped doing this in .18 (maybe .17 too, I didn't use that version very long)

Now with 1.9.19, same problems.

I tried to report the bug with .18 but github had some weird issue with issues.

I might have the free time to show off the problems in a video.

Windows 11 (21H2)

**Comment by clsid2** [comment]
The Add Containing Folder function was simplified. I might revert its behavior, but I need to understand how people are using it first.
You say you add one file, and then add the rest of the folder. You should know that you can also just drag&drop a folder.
But you seem to be moving the added MP4 file up in an existing playlist. Why?

Playlist still auto-hides for me in fullscreen with .19.

**Comment by harakiwi** [comment]
>You should know that you can also just drag&drop a folder.
But you seem to be moving the added MP4 file up in an existing playlist. Why?

I download videos, I watch them offline. Years ago I found MPC-HC because it was the only player that remembers the playback position of a file (accidentally skipping files is/was a problem with cats around the room).
The other feature I need is the option to delete files from the playlist. 
After I watched a video I may want to delete it. Saves a lot of time to delete it from the playlist and from my NAS in two clicks.

I create a playlist of videos from different sources. Twitch, Youtube, Floatplane, etc. Then a new video releases, sometimes multiple new videos. Then I want to watch those first, then resume my playlist.
By adding the first video from that folder, moving it up to the currently playing video (and Add Containing Folder) I don't have to manually move multiple videos from the bottom of the playlist.



I'll explain using music:

Let's say I have 10 albums worth of music. That's sometimes 20 - 25 songs per album, 200 - 250 songs, a very long playlist.
I add my files to the playlist. Drag the files around to manually mix some of the songs.

I didn't finish my playlist and want to continue the next day. This next day I see that a new album was downloaded overnight. I don't want to re-shuffle the whole playlist, just drag these new songs around to get some mild "randomness". 
I used to be able to drag one file to the middle, use Add Containing Folder, it adds the missing files in the middle of my playlist. 

With the current way it adds them at the bottom and I have to scroll up 10-20 times to get back to the middle.
It's way faster the old way.


>auto-hide

I use MPC-HC on my secondary monitor, to the right of my main monitor.

If I move my mouse to the left, from 2nd to main monitor the playlist stays open. I just noticed I have to move the mouse up, then left to have auto-hide working.
